config/initializers/secret_token.rb

```diff
@@ -4,4 +4,5 @@
 # If you change this key, all old signed cookies will become invalid!
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
-Faktory::Application.config.secret_token = '5011e3d1be36fdcd0daafc8c2dbe873b9a73a14ddd9369d9dd79d86e0a3d7fe5c95e1aebb7e9af5d3a4cc91e4715a9df7f4975e79e0242292efda3c91c413a3e'
+Faktory::Application.config.secret_token    = '5011e3d1be36fdcd0daafc8c2dbe873b9a73a14ddd9369d9dd79d86e0a3d7fe5c95e1aebb7e9af5d3a4cc91e4715a9df7f4975e79e0242292efda3c91c413a3e'
+Faktory::Application.config.secret_key_base = '5011e3d1be36fdcd0daafc8c2dbe873b9a73a14ddd9369d9dd79d86e0a3d7fe5c95e1aebb7e9af5d3a4cc91e4715a9df7f4975e79e0242292efda3c91c413a3e'
```

Comment:
Line is too long. [176/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.